Assets/LeapMotion/Core/Scripts/Attributes/ImplementsInterface.cs

```diff
@@ -55,16 +56,22 @@ public class ImplementsInterfaceAttribute : CombinablePropertyAttribute,
     /// the interface that this attribute constrains objects to, and returns the object
     /// that implements that interface, or null if none was found.
     /// </summary>
-    public UnityObject FindImplementer(UnityObject obj) {
+    public UnityObject FindImplementer(SerializedProperty property, UnityObject obj) {
```

Comment:
Why was the SerializedProperty argument added?  It isn't used anywhere?

Reply:
Hm, this might be a dangling argument then. Will fix if it's truly unused